docs/docs/parachains/composable/composable-council.md

```diff
@@ -0,0 +1,125 @@
+# A Look into the Composable Parachain's Governance
+
+Similar to the [governance of Picasso](../parachains/picasso/governance), Composable’s Polkadot parachain will ultimately become completely decentralized and democratic. This requires a sufficient user base, so in the initial stages of parachain deployment, the Composable Council will aid in governance. 
```

Comment:
> Composable’s Polkadot parachain will ultimately become completely decentralized and democratic.

Democracy v1 enabled and works. We just not use it.

Reply:
So we are democratic, not sure if decenralized (I mean about PICA distribution). Same may be about Centauri,